docs/image/README.md

```diff
@@ -0,0 +1,16 @@
+# Docs Test
+this container helps to test the documentation before you push to github
+
+## How-to
+
+```
+$ docker build -t local/deis-docs .
+$ docker run --rm -t -i -v `pwd`/../../../deis/deis:/deis -p 8000:8000 local/deis-docs
+```
+
+### Using a HTTP Proxy
```

Comment:
I don't think we need to call this out in the documentation. Users of docker will already know how to set this up on their own, and [docker will use your own proxy settings](https://coreos.com/docs/launching-containers/building/customizing-docker/#use-an-http-proxy) so this won't be necessary.